tello_terrain_follow_node:
  ros__parameters:
    target_distance: 2.0
    kp: 0.7
    ki: 0.1
    kd: 0.05
    max_vel: 5.0
    min_vel: -5.0
